### PR TITLE
Fix(web,web-twig): Fix `Button` flickering during spinner animation in Firefox

### PR DESCRIPTION
**Pull request by adamkudrna**
https://github.com/lmc-eu/spirit-design-system/assets/5614085/1acca4fd-b26d-4784-af7e-357608dcfba0

### CODE REVIEW

**Review by pavelklibani — APPROVED** [review]

File: packages/web-twig/src/Resources/components/Button/Button.twig [hunk 1]
```diff
@@ -33,6 +33,6 @@
 >
     {%- block content %}{% endblock %}
     {%- if _isLoading -%}
-        <Spinner />
+        <Spinner isReusable={ false } />
```
Comment (literat) [inline packages/web-twig/src/Resources/components/Button/Button.twig:36]: Please, do not forget to update the snapshot tests. They are failing now.

**Review by literat — CHANGES_REQUESTED** [review]

**Review by literat — APPROVED** [review]
Great 👍

File: packages/web/src/scss/components/Button/_Button.scss [hunk 2]
```diff
@@ -81,6 +83,7 @@
 
 .Button--loading {
     position: relative;
+    overflow: hidden; // 1.
```
Comment (crishpeen) [inline packages/web/src/scss/components/Button/_Button.scss:86]: hopefully, it won't blow up somewhere

**Review by crishpeen — APPROVED** [review]

File: packages/web/src/scss/components/Button/_Button.scss (hunk 2 above)
Comment (adamkudrna) [reply packages/web/src/scss/components/Button/_Button.scss:86]: The only use case I can think of is something [like this](https://react-ui.io/components/button#icon-buttons-with-a-badge). But we don't have that in Spirit.

image

**Pull request merged**